src/platforms/windows/app-host-clobbers.js

```diff
@@ -8,5 +13,17 @@ module.exports = {
         Application: {
             addEventListener: function () {}
         }
+    },
+    Windows: {
+        Networking: {
+            Connectivity: {
+                NetworkConnectivityLevel: {
```

Comment:
This is needed too by the network info plugin when loading `src/windows/NetworkInfoProxy.js` for `windows` platform. The reason I've added it here as a common API is because it is an enumeration defined that any other plugin could use. Since it is well-defined, there's no need that any simulation plugin re-define it again only for the plugin. This can be shared.